plugins/GSdx/GSRenderer.cpp

```diff
@@ -556,13 +556,23 @@ bool GSRenderer::MakeSnapshot(const string& path)
 {
 	if(m_snapshot.empty())
 	{
-		time_t t = time(NULL);
+		time_t cur_time = time(nullptr);
+		static time_t prev_snap;
+		static int n = 2;
```

Comment:
In the code. I guess to keep compatibility, there is no .1 image hence the .2 start.